src/main/java/seedu/taassist/ui/MainWindow.java

```diff
@@ -178,7 +178,7 @@ private void handleFocusMode() {
     }
 
     @FXML
-    private void handleUnfocusMode() {
+    private void handleUnfocusMode() throws CommandException, ParseException {
```

Comment:
`throws CommandException, ParseException` is unnecessary here. I guess it was added by IDE.